a3update.py

```diff
@@ -75,12 +77,29 @@ def log(msg: str):
     print(msg)
     print("{{0:=<{}}}".format(len(msg)).format(""))
 
+def count_sub_directories(rootPath: str):
+    counter = 0
+    for file in os.listdir(rootPath):
+        potentialDirectory = os.path.join(rootPath, file)
+        if(os.path.isdir(potentialDirectory)):
+            counter = counter+1
+    return counter
+
 
 def call_steamcmd(params: str):
     debug('steamcmd {}'.format(params))
     os.system("{} {}".format(STEAM_CMD, params))
     print("")
 
+def call_steamcmd_script(file: str):
```

Comment:
Not really seeing the benefit of using this over the `call_steamcmd` fn.

Reply:
I have lost the forum thread I read that gave me the idea.

What I remember reading was that when the mod download times out, it's actually continuing in the background.
When you give steam_cmd a new command, it stops the workshop_item_download and it goes on to the next one. 

So I thought, maybe if I go down the steamcmd script solution, that won't happen? 

I don't have any statistical data regarding what's more reliable between the 2 approaches, it was a blind attempt to fix the problem. For all I know, both might work the same.